resources/js/components/GlobalSlideover.vue

```diff
@@ -0,0 +1,31 @@
+<script>
+export default {
+    props: {
+        title: String,
+        position: String,
+        contents: String,
+    },
+
+    render() {
+        return this.$scopedSlots.default(this)
+    },
+
+    methods: {
+        open() {
+            this.$root.globalSlideover.content = this.slideoverContents
+            this.$root.globalSlideover.title = this.title
+            this.$root.globalSlideover.position = this.position
+        },
+    },
+
+    computed: {
+        slideoverContents() {
+            if (this.contents) {
+                return this.contents
+            }
+
+            return this.$el.querySelector('.global-slideover-wrapper').innerHTML
```

Comment:
Am i correct in seeing that this element does not exist in this PR?
Also note that this is not reactive, so updates to this elements content will not update the slideover

Reply:
yes, this is something that's useful for when/if we implement a mobile-only version of this. In that case, we need the content to actually exist in the DOM, and thus we would have to have a queryable div.

However, as for right now, that's not implemented, so it might be better to just leave this out for now?

Reply:
If we're not using it; remove it.